lib/publiccloud/gce.pm

```diff
@@ -18,15 +18,14 @@ use testapi;
 use strict;
 use utils;
 
-use constant CREDENTIALS_FILE => 'google_credentials.json';
-
 has account             => undef;
 has project_id          => undef;
 has private_key_id      => undef;
 has private_key         => undef;
 has service_acount_name => undef;
 has client_id           => undef;
-has storage_name        => undef;
+
+our $credentials_file = 'google_credentials.json';
```

Comment:
@cfconrad  WDYT ? should we keep it as constant or you OK with this change ?

Reply:
I would keep constant, but maybe there is a reason I do not know.

Reply:
ok